### PR TITLE
Add cancel order method

### PR DESCRIPTION
**Pull request by brunoroux**
Adding a new method to cancel an order according to : https://shopify.dev/docs/admin-api/rest/reference/orders/order?api[version]=2020-04

### CODE REVIEW

File: src/Services/Order.php [hunk 1]
```diff
@@ -159,6 +159,13 @@ public function createTest($order)
         return $this->unserializeModel($raw['order'], ShopifyOrder::class);
     }
 
+    public function cancel($id)
```
Comment (Wryte) [inline src/Services/Order.php:162]: Looks like the cancel endpoint supports passing in some data for refunds. Should we allow that in this method as well?

image

Comment (brunoroux) [reply src/Services/Order.php:162]: How should I go about adding that data in there ? 
Should I create a CancelOrder model ? 
And add that as an optional argument in the cancel method ?

Comment (Wryte) [reply src/Services/Order.php:162]: Yeah that would make the most sense to me. Just a plain array would be too difficult to know what is possible to set there.

Comment (brunoroux) [reply src/Services/Order.php]: I added the required Model

**Review by Wryte — APPROVED** [review]

**Pull request merged**